icu4j/main/classes/core/src/com/ibm/icu/text/PluralRanges.java

```diff
@@ -24,7 +24,7 @@
  * @deprecated This API is ICU internal only.
  */
 @Deprecated
-public final class PluralRanges implements Freezable<PluralRanges>, Comparable<PluralRanges> {
+final class PluralRanges implements Freezable<PluralRanges>, Comparable<PluralRanges> {
```

Comment:
FYI  I just checked the CLDR code, and I don't see any use of "ibm.icu.text.PluralRanges".